fdbclient/ServerKnobs.cpp

```diff
@@ -802,6 +802,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( EMPTY_READ_PENALTY,                                   20 ); // 20 bytes
 	init( DD_SHARD_COMPARE_LIMIT,                               1000 );
 	init( READ_SAMPLING_ENABLED,                                true ); if ( randomize && BUGGIFY ) READ_SAMPLING_ENABLED = false;// enable/disable read sampling
+	init( DD_TRACE_MOVE_BYTES_AVERAGE_INTERVAL,                   120);
+    init( DD_TRACE_MOVE_BYTES_SAMPLE_SIZE,                   10000000); // 10MB
```

Comment:
The size doesn't need to be specific to the DD move bytes case. We just need to protect the Deque from OOMing in general (and we don't expect to hit the size in normal case), so it's probably fine to just have a `MOVING_WINDOW_SAMPLE_SIZE` (or something like that) and use it directly in the `MovingWindow` class (rather than passing it in). This way, if we end up creating many different `MovingWindow`s in the future, and they're taking up a lot of memory, we just have to tweak one knob to bring down the memory use.

Reply:
That makes sense! Done